jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/comments/CaseCommentItemView.java

```diff
@@ -173,4 +201,79 @@ public void updateCommentPressingEnter(@ForEvent("keyup") final KeyboardEvent e)
         }
     }
 
+    @Override
+    public CaseCommentSummary getValue() {
+        return caseCommentSummary.getModel();
+    }
+
+    @Override
+    public void setValue(final CaseCommentSummary model) {
+        this.caseCommentSummary.setModel(model);
+        this.updateCommentText.setValue(model.getText());
```

Comment:
I guess this could be replaced by using `@Bound` annotation on updateCommentText.